GraphStructure/Main.java

```diff
@@ -0,0 +1,16 @@
+import java.util.ArrayList;
+
+public class Main{
```

Comment:
Shall move this into unit test. Try junit. In order to make your programming solid, try to create multiple test cases. For example, if you try to add edge with indices out of bound